external-crates/move/crates/move-vm-types/src/values/values_impl.rs

```diff
@@ -1420,13 +1419,6 @@ impl Value {
             RefCell::new(it.into_iter().collect()),
         ))))
     }
-
-    // REVIEW: This API can break
-    pub fn vector_for_testing_only(it: impl IntoIterator<Item = Value>) -> Self {
```

Comment:
I can add a wrapper `fn vector` that calls `Vector::pack`, but it seemed superfluous. Happy to do so if we want it though.

Reply:
I think having a separate API (as-is) for this is fine